srcs/python/quiver/feature.py

```diff
@@ -217,12 +198,10 @@ def from_cpu_tensor(self, cpu_tensor: torch.Tensor):
             cpu_tensor (torch.FloatTensor): input cpu tensor
         """
         if self.cache_policy == "device_replicate":
-            cache_memory_budget = self.cal_memory_budget_bytes(
-                self.device_cache_size)
+            cache_memory_budget = cal_memory_budget_to_bytes(self.device_cache_size)
```

Comment:
`cal_memory_budget_to_bytes(self.device_cache_size)` should be called once on construction.

Reply:
We only calculate once for this parse for each quiver.Feature